src/plugin/objectSupport/index.js

```diff
@@ -0,0 +1,96 @@
+export default (o, c) => {
+  const proto = c.prototype
+
+  /**
+   * Array to date
+   * @param d
+   * @param utc
+   * @returns {Date}
+   */
+  const parseArrayArgument = (d, utc) => {
+    if (utc) {
+      return new Date(Date.UTC(d[1], d[2] - 1, d[3]
+        || 1, d[4] || 0, d[5] || 0, d[6] || 0, d[7] || 0))
+    }
+    return new Date(d[1], d[2] - 1, d[3] || 1, d[4] || 0, d[5] || 0, d[6] || 0, d[7] || 0)
+  }
+
+  /**
+   * Converts the object to a date array
+   * @param d Object { years, months, date, hours, minutes, seconds, milliseconds}
+   * @param utc
+   */
+  const parseObjectArgument = (d, utc) => parseArrayArgument([
+    0,
+    d.y || d.year || d.years || 1970,
```

Comment:
check our `this.$utils().p()` prettyUnit to better handle this.

Reply:
Their role is different, `d` is input by the user
`d => { years: 2010, months: 1, days: 14, hours: 15, minutes: 25, seconds: 50, milliseconds: 125 }`
or
`d => { y: 2010, M: 1, d: 14, h: 15, m: 25, s: 50, ms: 125 }`
I don't have to judge units, just need to get the value of some units, just like the REGEX_PARSE in constant.js.
This code is similar to the part in index.js where parseDate handles strings.
` const d = date.match(C.REGEX_PARSE)`  In the index.js 60 lines

Reply:
https://github.com/iamkun/dayjs/blob/dev/src/plugin/duration/index.js#L42

Something like this.

Reply:
Is that right? But it doesn't support  date-> `dayjs({date:1})`

Reply:
1 you can get `prettyUnit` from `this.$utils().p` instead of important it directlly.

2 you may have to extend a new `prettyUnit` function to treat `day and date key both mean day-of-the-month.` like https://github.com/iamkun/dayjs/blob/dev/src/plugin/duration/index.js#L26

Reply:
Is that all right?

Reply:
cool

Reply:
Thank you, seniors, let me learn a lot.
This is my first PR. Thanks♪(･ω･)ﾉ

Reply:
You are most welcome, cheers.